scripts/build-controller-image.sh

```diff
@@ -59,11 +70,9 @@ if [[ ! -d $SERVICE_CONTROLLER_SOURCE_PATH ]]; then
     exit 1
 fi
 
-pushd $SERVICE_CONTROLLER_SOURCE_PATH 1>/dev/null
-
-SERVICE_CONTROLLER_GIT_VERSION=`git describe --tags --always --dirty || echo "unknown"`
-SERVICE_CONTROLLER_GIT_COMMIT=`git rev-parse HEAD`
-
+pushd "$SERVICE_CONTROLLER_SOURCE_PATH" 1>/dev/null
+  SERVICE_CONTROLLER_GIT_VERSION=$(git describe --tags --always --dirty || echo "unknown")
+  SERVICE_CONTROLLER_GIT_COMMIT=$(git rev-parse HEAD)
```

Comment:
Ah, i guess this is shellcheck recommendation?

Reply:
Yes. Using backticks to run commands is not recommended